src/styles.js

```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+    container:{
+        backgroundColor: '#f0f0f0',
+        alignItems:'center'
+    }
+})
+
+export default styles;
```

Comment:
Make sure you are repeating EOF mistakes again and again. Mistakes are fine, but repeated mistakes are not fine.